gui-daemon/xside.c

```diff
@@ -4242,6 +4336,18 @@ static void parse_vm_config(Ghandles * g, config_setting_t * group)
             exit(1);
         }
     }
+
+    if ((setting =
+         config_setting_get_member(group, "xbuf_max_delay"))) {
+        int delay_val = config_setting_get_int(setting);
+        if (delay_val < 0 || delay_val > 5000) {
+            fprintf(stderr,
+                    "unsupported value ‘%d’ for xbuf_max_delay (must be >= 0 and <= 5000)",
```

Comment:
```suggestion
                    "unsupported value '%d' for xbuf_max_delay (must be >= 0 and <= 5000)",
```

Smart quotes (`‘` and `’`) are better typographical practice, but Qubes OS generally uses ASCII for anything that gets logged.  This is because various parts of the codebase clobber non-ASCII bytes for security reasons.

Reply:
what the heck, I typed that in Vim...? Not sure how that became smart quotes, but 100% agreed on the change, will integrate into the next iteration.

Reply:
I see what happened, the similar "unsupported value" line just a bit above also used smart quotes. I probably copy-pasted that and used it as a template. Both places will be fixed in the next iteration.